projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -3169,6 +3178,97 @@ private void populateFlowHistory(FlowHistory flowHistory, String environmentName
     }
   }
 
+  // this is a hack to deal with router names that are named ethX
```

Comment:
Yes, that's what I meant.

The problem is that the normalization isn't applied uniformly -- it's applied to the cisco interface names, but not to host interface names.

My opinion is that the interface name normalization is a bad idea. Not only does it make it difficult to do matching, but it makes debugging more difficult also (like 'hm... why is my router interface Ethernet0 instead of eth0')